go-controller/pkg/config/config.go

```diff
@@ -1072,6 +1073,12 @@ var OvnKubeNodeFlags = []cli.Flag{
 		Value:       OvnKubeNode.MgmtPortNetdev,
 		Destination: &cliConfig.OvnKubeNode.MgmtPortNetdev,
 	},
+	&cli.StringFlag{
+		Name:        "disable-ovn-iface-id-ver",
+		Usage:       "if iface-id-ver option is not enabled in ovn, set this flag to True (depends on ovn version)",
```

Comment:
suggestion: list minimum OVN version in the Usage.

Reply:
In this case, it would be a git commit or 21.09 :)

Reply:
Well I can comment 21.09 version as minimal? I guess it's better than nothing

Reply:
yah some kind of useful version would be good.